tensorflow/python/ops/collective_ops_test.py

```diff
@@ -346,6 +347,30 @@ def testCollectiveGatherShapeMismatchAcrossDevices(self):
                                    'Shape mismatch'):
         sess.run([c0, c1], options=run_options)
 
+  @test_util.run_deprecated_v1
+  def testCollectiveGatherPolymorphicShape(self):
+    t0 = [0, 1, 2, 3, 4, 5, 6, 7]
+    t1 = [10, 11, 12, 13, 14, 15, 16, 17]
+    t01 = [0, 1, 2, 3, 4, 5, 6, 7, 10, 11, 12, 13, 14, 15, 16, 17]
+    t01_ = [1, 2, 3, 4, 5, 6, 7, 11, 12, 13, 14, 15, 16, 17]
+    group_size = 2
+    group_key = 1
+    instance_key = 123
+    with self.session(config=config_pb2.ConfigProto(device_count={'CPU': group_size})) as sess:
```

Comment:
`FAIL: Found 1 non-whitelisted pylint errors:
tensorflow/python/ops/collective_ops_test.py:359: [C0301(line-too-long), ] Line too long (95/80)`
can you please fix this ?

Reply:
lol cool sorry fixed